src/main/java/com/futurcollars/lesson8/zad4/FirstText.java

```diff
@@ -0,0 +1,9 @@
+package com.futurcollars.lesson8.zad4;
+
+public class FirstText implements TextFormatter {
```

Comment:
Nazwij proszę klase dokładniej, FIrstText nic nie mówi, wystarczy jak to nazwiesz UpperCaseTextFormatter